src/settings.js

```diff
@@ -383,13 +383,23 @@ const CssCollector = class CssCollector {
     if (!/^https?:\/\//i.test(url)) {
       throw new Error('Embedded urls should be absolute. ' + url)
     }
-    if (!(indexOf.call(this.urls, url) >= 0)) {
+    if (!(this.urls.includes(url))) {
       return this.urls.push(url)
     }
   }
 
   addStyle(style) {
-    return this.styles.push(style)
+    if (!(this.styles.includes(style))) {
+      return this.styles.push(style)
+    }
+  }
+
+  removeUrl(url) {
+    this.urls = this.urls.filter(item => item !== url)
+  }
+
+  removeStyle(style) {
+    this.urls = this.urls.filter(item => item !== style)
```

Comment:
Probably, there is a mistake

```suggestion
    this.styles = this.styles.filter(item => item !== style)
```